LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaWan.NetworkServer.csproj

```diff
@@ -26,6 +26,11 @@
   <ItemGroup>
     <AdditionalFiles Include="../../../../stylecop.json" Link="stylecop.json" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
probably not something you want to check in?

Reply:
I don't mind adding a class diagram. Removing for now.

Reply:
Me neither, but I think it was referencing a local path. Usually the trickiest part is to keep a checked in CD up to date.